docs/permissions/data-sandboxes-draft.md

```diff
@@ -0,0 +1,309 @@
+---
+title: Data sandboxes
+---
+
+# Data sandboxes
+
+{% include plans-blockquote.html feature="Data sandboxes" %}
+
+Data sandboxes let you give granular permissions to rows and columns for different groups of people.
+
+Say you have people who you want to be able to log into your Metabase, but who should only be able to view data that pertains to them. For example, you might want to display an Accounts table to some customers, but each customer should only be able to view the rows that match their customer ID. Sandboxes let you give people a way to self-serve their own analytics, without ever seeing any results that don't pertain to them.
```

Comment:
Maybe link to https://www.metabase.com/learn/customer-facing-analytics/multi-tenant-self-service-analytics for context.